tcptracer-bpf.h

```diff
@@ -79,16 +79,23 @@ struct tracked_socket {
 
 #define EVENT_HTTP_RESPONSE 1
 #define EVENT_MYSQL_GREETING 2
+#define EVENT_HTTP_RESPONSE_V6 3
+#define EVENT_MYSQL_GREETING_V6 4
+
+union connections {
```

Comment:
Maybe it makes more sense to move the type of the connection to a `connection_wrapper` or something like this. Then you have only one type of `EVENT_HTTP_RESPONSE` and `EVENT_MYSQL_GREETING` and you don't have to repeat a lot of code.